compiler/src/main/scala/org/scalajs/core/compiler/JSGlobalAddons.scala

```diff
@@ -108,13 +108,13 @@ trait JSGlobalAddons extends JSDefinitions
 
     /** has this symbol to be translated into a JS getter (both directions)? */
     def isJSGetter(sym: Symbol): Boolean = {
-      sym.tpe.params.isEmpty && enteringUncurryIfAtPhaseAfter {
+      sym.isModule || (sym.tpe.params.isEmpty && enteringUncurryIfAtPhaseAfter {
```

Comment:
we only get here when `sym.isMethod`, thus `sym.isModule` can only hold for the module's accessor